wallet-connect/src/v2/client.rs

```diff
@@ -219,6 +222,22 @@ impl<M: Middleware> MiddlewareError for WCError<M> {
     }
 }
 
+fn pad_zero(s: String) -> String {
```

Comment:
We could add more documentation to the functions, and describe the params.

Reply:
ok